app/Http/Controllers/API/NotificationController.php

```diff
@@ -3,43 +3,39 @@
 namespace App\Http\Controllers\API;
 
 use App\Http\Controllers\NotificationController as NotificationBackend;
+use Illuminate\Http\JsonResponse;
 
 class NotificationController extends ResponseController
 {
     /**
      * Display a listing of the resource.
      *
-     * @return \Illuminate\Http\Response
+     * @return JsonResponse
      */
-    public function index()
-    {
+    public function index() {
         $notificationResponse = NotificationBackend::latest();
         return $this->sendResponse($notificationResponse);
     }
 
     /**
      * Update the specified resource in storage.
      *
-     * @param  \Illuminate\Http\Request $request
-     * @param  int                      $notificationID
+     * @param int $notificationID
      *
-     * @return \Illuminate\Http\Response
+     * @return JsonResponse
      */
-    public function update($notificationID)
-    {
+    public function update(int $notificationID) {
         return NotificationBackend::toggleReadState($notificationID);
     }
 
     /**
      * Remove the specified resource from storage.
      *
-     * @param  int $notificationID
+     * @param int $notificationID
      *
-     * @return \Illuminate\Http\Response
+     * @return JsonResponse
```

Comment:
Can you add ``JsonResponse`` to the function, so it become Typesafe? (Just a wish from my side.)